### ISSUE TITLE
Attempt to submit private repository 

### ISSUE THREAD

**Issue by norberttech** [composer/packagist#1456]
Hey! 
I accidentally created a private repository and then when I tried to submit them I got this error: 

![image](https://github.com/composer/packagist/assets/1921950/8dcb9d1b-60e2-4723-afdf-11e72193c632)

Which makes me think that a bit too much is being exposed, in case of more severe issues, some sensitive information might be leaked out.

**Comment by Seldaek** [comment]
Yeah, we don't really have much to hide in that sense there, as it is all public and open source. I realize there is some risk involved but I much prefer to carry that risk than output generic error messages that are not helpful in debugging issues.
